sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroIO.java

```diff
@@ -806,9 +806,8 @@ abstract Builder<T> setFileExceptionHandler(
 
       abstract ReadFiles<T> build();
     }
-
-    @VisibleForTesting
-    ReadFiles<T> withDesiredBundleSizeBytes(long desiredBundleSizeBytes) {
+    
```

Comment:
Thanks, seems it is now exposed to user, would you mind adding a brief javadoc? e.g. information in your PR description addressing when this may be useful.

Reply:
I will be adding this asap, thanks for the tip!

Reply:
I have added Javadoc strings to all the methods that are now public.